src/System.Reflection.Emit.ILGeneration/tests/Generator.cs

```diff
@@ -0,0 +1,427 @@
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+
+namespace TestLibrary
+{
+    public static class Generator
+    {
+        internal static Random m_rand = new Random();
+        internal static int? seed = null;
+
+        public static int? Seed
+        {
+            get
+            {
+                if (seed.HasValue)
+                {
+                    return seed.Value;
+                }
+                else
+                {
+                    return null;
+                }
+            }
+            set
+            {
+                if (!(seed.HasValue))
+                {
+                    seed = value;
+                    if (seed.HasValue)
+                    {
+                        m_rand = new Random(seed.Value);
+                    }
+                }
+            }
+        }
+
+        // returns a byte array of random data
+        public static void GetBytes(int new_seed, byte[] buffer)
+        {
+            Seed = new_seed;
+            GetBytes(buffer);
+        }
+        public static void GetBytes(byte[] buffer)
+        {
+            m_rand.NextBytes(buffer);
+        }
+
+        // returns a non-negative Int64 between 0 and Int64.MaxValue
+        public static Int64 GetInt64(int new_seed)
+        {
+            Seed = new_seed;
+            return GetInt64();
+        }
+        public static Int64 GetInt64()
+        {
+            byte[] buffer = new byte[8];
+            Int64 iVal;
+
+            GetBytes(buffer);
+
+            iVal = 0;
+            for (int i = 0; i < buffer.Length; i++)
+            {
+                iVal |= ((Int64)buffer[i] << (i * 8));
+            }
+
+            if (0 > iVal) iVal *= -1;
+
+            return iVal;
+        }
+
+        // returns a non-negative int between 0 and int.MaxValue
+        public static int GetInt32(int new_seed)
+        {
+            Seed = new_seed;
+            return GetInt32();
+        }
+        public static int GetInt32()
+        {
+            int i = m_rand.Next();
+            return i;
+        }
+
+        // returns a non-negative Int16 between 0 and Int16.MaxValue
+        public static Int16 GetInt16(int new_seed)
+        {
+            Seed = new_seed;
+            return GetInt16();
+        }
+        public static Int16 GetInt16()
+        {
+            Int16 i = Convert.ToInt16(m_rand.Next() % (1 + Int16.MaxValue));
+            return i;
+        }
+
+        // returns a non-negative Byte between 0 and Byte.MaxValue
+        public static Byte GetByte(int new_seed)
+        {
+            Seed = new_seed;
+            return GetByte();
+        }
+        public static Byte GetByte()
+        {
+            Byte i = Convert.ToByte(m_rand.Next() % (1 + Byte.MaxValue));
```

Comment:
Does the code formatter not change usage of the type name to the keyword for primities, e.g. byte instead of Byte?